x-pack/plugins/uptime/server/lib/synthetics_service/formatters/format_configs.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { isNil, omitBy } from 'lodash';
+import { ConfigKey, MonitorFields } from '../../../../common/runtime_types/monitor_management';
+import { formatters } from './index';
+
+const UI_KEYS = [
```

Comment:
I think this is unnecessary. The formatters themselves determine what needs to be formatted. I feel like this creates a bit of unnecessary repition of logic.

Reply:
I misunderstood what this array was for. This implementation is fine by me.